ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDInterestType.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.cmc.ccd.domain;
+
+public enum CCDInterestType {
```

Comment:
same as above? Do we have to create a separate set of enums? They should be the same, right?

Reply:
they are based on ccd definition excel sheet